acceptance_tests/features/pages/collection_exercise.py

```diff
@@ -0,0 +1,21 @@
+from acceptance_tests import browser
+from config import Config
+
+
+def go_to(survey, period):
+    browser.visit(f'{Config.RESPONSE_OPERATIONS_UI}/surveys/{survey}/{period}')
```

Comment:
This should be consistent with the .format method used in the repo

Reply:
Arguably, an f-string is much more readable than `.format` here (and for URI building in general).